ui/imports/shared/popups/addaccount/panels/WatchOnlyAddressSection.qml

```diff
@@ -6,6 +6,8 @@ import StatusQ.Core.Theme 0.1
 import StatusQ.Controls 0.1
 import StatusQ.Controls.Validators 0.1
 
+import StatusQ.Core.Backpressure 0.1
+
 import utils 1.0
 
 import "../stores"
```

Comment:
remove imports with relative paths.

Reply:
No need for this, since the store is internal for the add account popup, not exposed, not maintained outside.

Reply:
yes the problem will happen when we create storybook page and Unit tests, relative path give issues when unit testing the qml, again not something to be done in this PR, but all relative imports should be avoided.